src/ansys/aedt/core/application/analysis.py

```diff
@@ -782,8 +783,12 @@ def list_of_variations(self, setup=None, sweep=None):
                 return [""]
 
     @pyaedt_function_handler()
+    @deprecate_argument(
+        "analyze", "The ``analyze`` argument will be deprecated in future versions." "Analyze before exporting results."
```

Comment:
```suggestion
        "analyze", "The ``analyze`` argument will be deprecated in future versions. Analyze before exporting results."
```
To be performed on other calls too.